packages/apputils/src/toolbar.tsx

```diff
@@ -244,6 +244,66 @@ export class Toolbar<T extends Widget = Widget> extends Widget {
     return true;
   }
 
+  /**
+   * Insert an item into the toolbar at the after a target item.
+   *
+   * @param at - The target item to insert after.
+   *
+   * @param index - The index at which to insert the item.
```

Comment:
This item in the docstring should be removed.

Reply:
Oops 😝. Good catch.